packages/docs-site/src/content/docs/docs/guides/schema.mdx

```diff
@@ -15,9 +9,9 @@ If you haven't already, follow our [Quickstart](/docs/get-started/quickstart) gu
 
 {/* TODO add link to example repo? */}
 
-## Project Structure
+## Project structure
 
-Using DMNO, we define a "configuration schema" for your entire project. We break that configuration up to live alongside each of the services in your code, and provide ways for them to reference each other as needed. Each service config is defined in a `config.mts` (typescript) file that lives in a `.dmno` directory.
+Using DMNO, we define a "configuration schema" for your entire project. We break that configuration up to live alongside each of the services in your code, and provide ways for them to reference each other as needed. Each service config is defined in a `config.mts` Typescript file that lives in a `.dmno` directory.
```

Comment:
looks like the official way is `TypeScript`

Reply:
oops, will update